### PR TITLE
New Cheat sheet Microsoft Powerpoint 2013

### PR DESCRIPTION
**Pull request by mayank7**
New Cheat sheet for Microsoft Powerpoint 2013 containing keyboard shortcuts.

IA Page : https://duck.co/ia/view/microsoft_powerpoint_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Microsoft Powerpoint](https://duck.co/ia/view/microsoft_powerpoint)

**Description**: Keyboard shortcuts for Microsoft Powerpoint 2013

**Example Query**: [https://duckduckgo.com/?q=microsoft+powerpoint&ia=about](https://beta.duckduckgo.com/?q=https://duckduckgo.com/?q=microsoft+powerpoint&ia=about), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: https://support.office.com/en-us/article/Use-keyboard-shortcuts-to-create-your-presentation-EBB3D20E-DCD4-444F-A38E-BB5C5ED180F4

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/microsoft_powerpoint)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/microsoft_powerpoint)*

File: share/goodie/cheat_sheets/json/microsoft-powerpoint.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                "val": "Select one line down",
+                "key": "[Ctrl] [↓]"
+            },
+            {
+                "val": "Select an object when the text inside the object is selected",
+                "key": "[Esc]"
+            },
+            {
+                "val": "Select text within an object with an object selected",
+                "key": "[Enter]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/microsoft-powerpoint.json]: @mayank7 please remove the `[]` around any single keys.

If you look at the rendered cheat sheet you'll see the `[]` in the actual content which isn't correct

**Comment by mayank7** [comment]
@moollaza I removed square brackets for single keys.

**Comment by mayank7** [comment]
@mintsoft I removed square brackets for single keys.

**Comment by mintsoft** [comment]
Thanks @mayank7

**Pull request merged**